.github/workflows/CD.yaml

```diff
@@ -250,14 +250,17 @@ jobs:
     if: "github.event_name == 'push' && (github.ref == 'refs/heads/main' || github.ref == 'refs/heads/alpha') && !contains(github.event.head_commit.message, 'chore(release):')"
     needs: [unit-tests, visual-tests, types, lint, lint-css, betterer, build]
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: ['12', '14', '16']
```

Comment:
This is not the place though :) the correct step start on line 210. This would cause error in better case and three releases in worse case